manticore/platforms/linux.py

```diff
@@ -1921,24 +1932,38 @@ def sys_exit_group(self, error_code):
         return self._exit(f"Program finished with exit status: {ctypes.c_int32(error_code).value}")
 
     def sys_ptrace(self, request, pid, addr, data):
+        logger.warning("Unimplemented system call: sys_ptrace")
         return 0
 
     def sys_nanosleep(self, req, rem):
+        logger.warning("Unimplemented system call: sys_nanosleep")
         return 0
 
     def sys_set_tid_address(self, tidptr):
         return 1000  # tha pid
 
     def sys_faccessat(self, dirfd, pathname, mode, flags):
+        logger.warning("Unimplemented system call: sys_faccessat")
         filename = self.current.read_string(pathname)
         return -1
 
     def sys_set_robust_list(self, head, length):
+        logger.warning("Unimplemented system call: sys_set_robust_list")
         return -1
 
-    def sys_futex(self, uaddr, op, val, timeout, uaddr2, val3):
+    def sys_sysinfo(self, infop):
+        logger.warning("Unimplemented system call: sys_sysinfo")
+        return -1
+
+    def sys_madvise(self, infop):
+        logger.warning("Unimplemented system call: sys_madvise")
         return -1
 
+    def sys_futex(self, uaddr, op, val, timeout, uaddr2, val3):
+        logger.warning("sys_futex isn't really implemented")
+        logger.debug(f"Futex at: {hex(uaddr)} -- {op}:{val} ({self.current.read_int(uaddr)})")
```

Comment:
This should not use f-string literals - as the output log string is produced by us (and so having a logging level that doesn't include debug will still waste time formatting the string).